pages/common/routers/multi-step.js

```diff
@@ -41,17 +41,14 @@ module.exports = ({ schema, config, root, postData = (req, res, next) => next()
   });
 
   app.use((req, res, next) => {
-    const step = req.step;
     const allSteps = Object.keys(config);
 
-    const previousSteps = allSteps.slice(0, allSteps.indexOf(step));
-
-    const includedSteps = previousSteps.filter(s => {
+    const steps = allSteps.filter(s => {
       const include = config[s].include;
       return !include || include(req);
```

Comment:
This implies that setting `include: false` will cause a step be be included. Seems odd.

Reply:
Sorry for being "commenting on white lines in the PR guy"